samples/CallWithChat/src/app/views/CallScreen.tsx

```diff
@@ -79,14 +79,19 @@ export const CallScreen = (props: CallScreenProps): JSX.Element => {
     [callIdRef]
   );
 
+  const callAdapterOptions = useMemo(() => {
```

Comment:
conditional compile this?

Reply:
(can leave if it doesn't cause a build failure)

Reply:
added the cc comment. 😊